cpp/models/abm/model.h

```diff
@@ -622,6 +596,32 @@ class Model
                            m_contact_exposure_rates_cache[person.get_location().get()], t, dt, parameters);
     }
 
+    /**
+     * @brief Implementation of Model::get_person. 
+     * This function needs to use a template as otherwise std::is_const would be false.
+     * @param[in] m Model in which the person_id is searched for.
+     * @param[in] person_id A Person's PersonId.
+     * @return A reference to the Person.
```

Comment:
```suggestion
     * This function needs to use a template to deduce whether the model and returned person should be const.
     * @param[in] m A reference to `*this`, so we can access m_persons.
     * @param[in] person_id A Person's PersonId.
     * @return A reference to the Person with matching ID.
```
I think this can be more specific